src/js/wall/LoginForm.tsx

```diff
@@ -30,23 +29,32 @@ const LoginContainer = styled.div`
 type LoginFormProps = {
     /** Error message for the login process. */
     error: string;
-    /** Callback to handle login submission */
-    onLogin: (username: string, password: string, remember: boolean) => void;
 };
 
 /** Handles the user login process. */
-export function LoginForm({ error, onLogin }: LoginFormProps) {
+export function LoginForm({ error }: LoginFormProps) {
```

Comment:
```suggestion
export default function LoginForm({ error }: LoginFormProps) {
```
Without needing to connect to redux, we can directyl export this component as default

Also does it still make sense to get the error from props? The redux implementation derives the text of the error from the message returned from the API. I think it would make sense to do the same thing here in the mutation. Good chance react-query has some nice support for doing this. If not you can use `onError` to handle it yourself